go.mod

```diff
@@ -4,6 +4,8 @@ go 1.17
 
 require github.com/stretchr/testify v1.7.1
 
+require github.com/go-sql-driver/mysql v1.7.0 // indirect
```

Comment:
I think we need 1.5.0 for backward compat
https://github.com/sansecio/ecomscan/blob/0fe0aa76fe597e7b5a66b51e40e87eaf3d45789c/go.mod#L12